### PR TITLE
Allow exiting the infinite loop using quit or exit

### LINKED ISSUE #4: Gracefully exit after fmdpy prompt
Wondering what is the best way to gracefully exit after the `fmdpy:` prompt. Otherwise, anything that you enter after the `fmdpy` gives some error 
```
fmdpy: asldkfjalsdkfjal;sdfjlas
Traceback (most recent call last):
  File "/home/arsaboo/.local/bin/fmdpy", line 8, in <module>
    sys.exit(fmdpy())
  File "/home/arsaboo/.local/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/fmdpy/__main__.py", line 121, in fmdpy
    c_pool.append(int(indx)-1)
ValueError: invalid literal for int() with base 10: 'asldkfjalsdkfjal;sdfjlas'
```

### PR DESCRIPTION
**Pull request by arsaboo**
Fixes #4

### CODE REVIEW

**Comment by Liupold** [comment]
Everything looks excellent, please make the requested changes, I would be happy to merge. Thank you for this pull request.

File: fmdpy/__main__.py [hunk 1]
```diff
@@ -93,6 +95,9 @@ def fmdpy(count, search, fmt, bitrate, multiple,
 
             prompt_input = input("\nfmdpy: ")
 
+            if prompt_input in ['quit', 'exit']:
```
[the remark this answers is not in the document]
Comment (arsaboo) [reply fmdpy/__main__.py]: > Everything looks excellent, please make the requested changes, I would be happy to merge. Thank you for this pull request.

Done.

**Comment by arsaboo** [comment]
Excellent....can you please update the release, so that it can be updated easily. Thanks!

**Comment by Liupold** [comment]
Done! https://github.com/Liupold/fmdpy/releases/tag/0.5.2. (Pypi should be updated by now.)

**Pull request merged**